docs/guides/Master-Data/master-data-how-it-works/vtable.md

```diff
@@ -232,7 +310,6 @@ VTable parses the JSON Schema configuration and each field to a corresponding UI
 
 To render a checkbox the field must have type **boolean**.
```

Comment:
Corrigir todas essas descrições, incluindo a vírgula apropriadamente:

To render a checkbox`,` the field must have type **boolean**.

To render a `DatePicker` `,` the field must have the property `format` with the value `date-time`.

....


Outra sugestão seria reescrever essas descrições, usando ações, exemplo:

```
Set the value of the `type` field as `boolean` to render a checkbox.
```